brain/circuit.cpp

```diff
@@ -215,6 +230,12 @@ class MVD3 : public Circuit::Impl
         , _circuit( config.getCircuitSource().getPath( ))
     {}
 
+    size_t getNumNeurons() const final
+    {
+        // Dirty cast until ::MVD3::MVD3File::getNbNeuron becomes const
```

Comment:
Indeed :-) I understood Adrien was on it last week so I let him do. I'll check with him what the status currently is.